spec/integration/compose/service_stacks_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "spec_helper"
+
+RSpec.describe "Compose service stacks" do
+  service_names ||= Dir.glob("*", base: "services")
+
+  service_names.each do |service_name|
+    it "configures #{service_name} with a lite or app stack" do
+      expect(compose_services(service_name).keys).to include(/lite/)
+        .or include(/app(-\w+)?$/)
+    end
+  end
+
+  compose_files = Dir.glob("services/**/docker-compose.yml")
+
+  compose_app_services = compose_files.flat_map do |filename|
```

Comment:
This definition is repeated in rails_debugger_spec and virtual_hosts_spec. Is it time to create a shared helper file for our tests? It would help to make this file more readable - current structure does not make very clear what is an expectation block and what is a 'setup' block.